### ISSUE TITLE
gradle clean eclipseFactorypath keeps adding existing entries does not deletes old one

### ISSUE THREAD

**Issue by balapal** [tbroyer/gradle-apt-plugin#90]
If I remove sg from the annotationProcessor configuration I have to manually delete the .factorpath file and run eclipseFactorypath reflect the change.

This is due to the fact that this gradle task leaves the existing entries intact and only appends the list of jars with the ones currently on the list (even if they are already listed)

Here is an example after multiple runs with the same two jars being added each time:
 ```
<?xml version="1.0" encoding="UTF-8"?>
 <factorypath>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate\hibernate-jpamodelgen\5.3.6.Final\4b35434c7b3c1228634bd1f7b0b445dcfaba3a21\hibernate-jpamodelgen-5.3.6.Final.jar" enabled="true" runInBatchMode="false"/>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate.javax.persistence\hibernate-jpa-2.0-api\1.0.1.Final\3306a165afa81938fc3d8a0948e891de9f6b192b\hibernate-jpa-2.0-api-1.0.1.Final.jar" enabled="true" runInBatchMode="false"/>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate\hibernate-jpamodelgen\5.3.6.Final\4b35434c7b3c1228634bd1f7b0b445dcfaba3a21\hibernate-jpamodelgen-5.3.6.Final.jar" enabled="true" runInBatchMode="false"/>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate.javax.persistence\hibernate-jpa-2.0-api\1.0.1.Final\3306a165afa81938fc3d8a0948e891de9f6b192b\hibernate-jpa-2.0-api-1.0.1.Final.jar" enabled="true" runInBatchMode="false"/>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate\hibernate-jpamodelgen\5.3.6.Final\4b35434c7b3c1228634bd1f7b0b445dcfaba3a21\hibernate-jpamodelgen-5.3.6.Final.jar" enabled="true" runInBatchMode="false"/>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate.javax.persistence\hibernate-jpa-2.0-api\1.0.1.Final\3306a165afa81938fc3d8a0948e891de9f6b192b\hibernate-jpa-2.0-api-1.0.1.Final.jar" enabled="true" runInBatchMode="false"/>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate\hibernate-jpamodelgen\5.3.6.Final\4b35434c7b3c1228634bd1f7b0b445dcfaba3a21\hibernate-jpamodelgen-5.3.6.Final.jar" enabled="true" runInBatchMode="false"/>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate.javax.persistence\hibernate-jpa-2.0-api\1.0.1.Final\3306a165afa81938fc3d8a0948e891de9f6b192b\hibernate-jpa-2.0-api-1.0.1.Final.jar" enabled="true" runInBatchMode="false"/>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate\hibernate-jpamodelgen\5.3.6.Final\4b35434c7b3c1228634bd1f7b0b445dcfaba3a21\hibernate-jpamodelgen-5.3.6.Final.jar" enabled="true" runInBatchMode="false"/>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate.javax.persistence\hibernate-jpa-2.0-api\1.0.1.Final\3306a165afa81938fc3d8a0948e891de9f6b192b\hibernate-jpa-2.0-api-1.0.1.Final.jar" enabled="true" runInBatchMode="false"/>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate\hibernate-jpamodelgen\5.3.6.Final\4b35434c7b3c1228634bd1f7b0b445dcfaba3a21\hibernate-jpamodelgen-5.3.6.Final.jar" enabled="true" runInBatchMode="false"/>
 	<factorypathentry kind="EXTJAR" id="C:\Users\Balazs\.gradle\caches\modules-2\files-2.1\org.hibernate.javax.persistence\hibernate-jpa-2.0-api\1.0.1.Final\3306a165afa81938fc3d8a0948e891de9f6b192b\hibernate-jpa-2.0-api-1.0.1.Final.jar" enabled="true" runInBatchMode="false"/>
 </factorypath>
```

**Comment by tbroyer** [comment]
Fwiw, I just published version 0.19 with this fix

**Comment by balapal** [comment]
looks good.
no duplicate entries anymore.
Thanks
